app/policies/presentation_policy.rb

```diff
@@ -6,4 +6,12 @@ def create?
   def new?
     create?
   end
+
+  def update?
+    (person&.admin? || person&.presentations&.include?(record)).present?
```

Comment:
Why not make the initializer a little smarter. 

https://github.com/StemboltHQ/staff_tracker/blob/master/app/policies/application_policy.rb#L5

This could become:
`@person = person || Person.new`

Then your're always guaranteed to have a person model to operate on